packages/next/src/server-build.ts

```diff
@@ -2449,6 +2527,26 @@ export async function serverBuild({
           ]
         : []),
 
+      // When PPR is not enabled but client segment cache is, then rewrite any
```

Comment:
what logic below is determining that PPR is not enabled?

Reply:
That was an old comment, updating!